### ISSUE TITLE
.NET 5 ClickOnce creates duplicate deployment in PublishDir

### ISSUE THREAD

**Issue by fuglede** [dotnet/deployment-tools#198]
This is a mirror of [this SO question](https://stackoverflow.com/q/71941042/5085211); figured it might be possible to get some eyes on it here as well.

### Description

When I published a ClickOnce application prior to the advent of .NET 5, I would generally expect to see files like

```
publishpath\setup.exe
publishpath\Application Files\MyApp_1_0_0_9\SomeLibrary.dll.deploy
```

When I use the .NET 5 version, all assemblies appear to be duplicated in the root of the publish directory as well:

```
publishpath\SomeLibrary.dll
```

### Regression?

Since this didn't happen on .NET Framework, I would imagine the files are redundant. But maybe not?

**Comment by NikolaMilosavljevic** [comment]
@sujitnayak @John-Hart - is this a known issue in VS ClickOnce publishing?

**Comment by John-Hart** [comment]
Yes, we're working on a better solution for this, you can track it https://developercommunity.visualstudio.com/t/Clickonce-for-NET-Core-issues/1500772.

**Comment by fuglede** [comment]
If it makes a difference, this also happens outside of VS, using the msbuild command line tool.

**Comment by HughWarrington** [comment]
Still a problem, even though https://developercommunity.visualstudio.com/t/Clickonce-for-NET-Core-issues/1500772 says the issue is fixed and released.

**Comment by sujitnayak** [comment]
@HughWarrington please use the Developer Community ticket for further follow-up.
